web-app/app/tricorder/page.tsx

```diff
@@ -100,11 +90,7 @@ function Tricorder(props: any) {
       <button
         className="m-4 p-4 rounded-full bg-red-900 font-bold text-white"
         onClick={() => {
-          const newFoundLifeforms = foundLifeforms;
-          newFoundLifeforms.map((_: any, idx: string) => {
-            delete newFoundLifeforms[idx as any as number];
-          });
-          setFoundLifeForms(newFoundLifeforms);
+          setFoundLifeForms([]);
```

Comment:
While I thought it was funny to delete members of the array, thus having our array indexing get all sorts of thrown off, this is a much saner solution.